pytorch/Dockerfile

```diff
@@ -109,6 +109,57 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+FROM multinode AS hf-genai
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends --fix-missing \
+    google-perftools \
+    libjemalloc2 \
+    libomp-dev \
+    numactl \
+    git
+
+COPY hf-genai-requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r hf-genai-requirements.txt
+
+# Install transformers from source to provide support for transformers example scripts
+RUN git clone https://github.com/huggingface/transformers.git && \
+    python -m pip install --no-cache-dir /transformers && \
+    python -m pip install --no-cache-dir -r /transformers/examples/pytorch/language-modeling/requirements.txt && \
+    rm -rf transformers
+
+# Install OpenSSH for MPI to communicate between containers
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
```

Comment:
You don't need the update because you didn't cleanup from the last apt command

Reply:
I'd fix the previous `RUN apt-get` command(s) and add the cleanup to them.
Each `RUN apt-get` command should end up with a cleanup to reduce the layer size if there's a chance for that.
If you did that, you want to leave this one unchanged. This keeps your `RUN` commands more consistent.

Reply:
Removed